.gitmodules

```diff
@@ -8,6 +8,7 @@
 [submodule "external/linker"]
     path = external/linker
     url = ../../mono/linker.git
+    branch = d15-4
```

Comment:
This looks wrong (in fact it doesn't look like there's a `d15-5` branch in the linker repo).